scss-partials/_general.scss

```diff
@@ -0,0 +1,265 @@
+
+    &:hover { text-decoration: underline; }
+}
+
+.create {
+    background: $tws_blue;
+    color: $tws_white;
+    text-transform: uppercase;
+    width: 94px;
+    padding: 4px 16px;
+    margin-right: 32px;
+
+    &:hover { background: $tws_coral; }
+
+    &__list {
+        text-align: center;
+        font-weight: bold;
+        letter-spacing: 0.4px;
+    }
+}
+
+.create, .findCouple {
+    display: none;
+
+    @include media ( ">=lg" ) {
+        display: inline-block;
+    }
+}
+
+.menu {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 0;
+    font-family: $font-regular;
+
+    @include media ( "<lg" ) {
+        display: none;
+    }
+
+    &__list {
+        display: flex;
+        justify-content: space-evenly;
+        width: 100%;
+        padding: 0px;
+        margin: 0px;
+    
+        &--item {
+            margin: 32px 8px 0px;
+            padding: 0px 0px 8px;
+            color: $tws_black;
+            border-bottom: 2px solid $tws_white;
+            letter-spacing: 0.8px;
+            white-space: nowrap;
+            list-style: none;
+            line-height: 20px;
+            cursor: pointer;
+    
+            &:hover { border-bottom: 2px solid $tws_coral; }
+        }
+    }
+}
+
+.search__img {
+    cursor: pointer;
+    height: 15px;
+    width: 15px;
+    margin: 17px 0px 7px 17px;
+
+    @include media ( ">=md" ) {
+        margin: 21px 0px 7px 17px;
+    }
+    
+    @include media ( ">=lg" ) {
+        margin: 33px 0px 16px 33px;       
+    }
+}
+
+.cta {
+    align-items: center;
+    text-align: center;
+    margin: 100px 0px 48px;
+
+    @include media ( "<=sm" ) {
+        margin: 64px 0px 0px;
+    }
+    
+    &__header {
+        color: $tws_black;
+        font-family: $font-heading;
+        font-style: normal;
+        font-size: 24px;
+        font-weight: 300;
+        line-height: 32px;
+        margin: 0px 16px;
+
+        @include media ( ">=md" ) {
+            font-size: 32px;
+            line-height: 40px;
+        }
+    }
+
+    &__info {
+        
+        &-text {
+            color: $tws_black;
+            font-family: $font-regular;
+            font-size: 16px;
+            line-height: 24px;
+            font-weight: 400;
+            max-width: 343px;
+            margin: 0 auto;
+            margin-top: 16px;
+            
+            @include media ( ">=md" ) {
+                max-width: 600px;
+                // previously had this as 588px as per the figma design but on live the 'journey' in the final sentence is on the third line not under it - where should this be?
+            }
+        }
+
+        &-btn {
+            font-family: $font-regular;
+            text-transform: uppercase;
+            color: $tws_white;
+            background-color: $tws_blue;
+            font-size: 14px;
+            line-height: 24px;
+            font-weight: 700;
+            margin: 32px 0px 0px;
+            padding: 16px 40px;
```

Comment:
I think you used styles for a larger button while it needs to be a bit smaller, check the prod version

Reply:
Not sure if I'm looking on the wrong page but I think what I've done matches the figma? 
![image](https://user-images.githubusercontent.com/113596529/213162292-ad1d0068-71d0-4de0-8709-a1c8a3e4d879.png)